Cargo.toml

```diff
@@ -113,7 +113,7 @@ actix-web-extras = "0.1.0"
 [target.'cfg(target_os = "linux")'.dependencies]
 rstack-self = { version = "0.3.0", optional = true }
 
-[target.'cfg(not(target_env = "msvc"))'.dependencies]
+#['cfg(all(not(target_env = "msvc"), any(target_arch = "x86_64", target_arch = "aarch64")))'.dependencies]
```

Comment:
@abhishek818 your logic is rather convoluted:
```py
all(not(target_env = "msvc"), any(target_arch = "x86_64", target_arch = "aarch64"))
```
as the first part becomes redundant by adding the second part. This is enough:
```py
any(target_arch = "x86_64", target_arch = "aarch64")
```

Reply:
same goes for the other instances of your logic